tests/report_tests/plugin_tests/curtin/curtin.py

```diff
@@ -0,0 +1,56 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos_tests import StageOneReportTest, StageTwoReportTest
+
+
+class CurtinPluginTest(StageOneReportTest):
+    """Basic sanity check to make sure common config files are collected
+
+    :avocado: tags=stageone
+    """
+
+    sos_cmd = '-o curtin'
+
+    def test_curtin_confs_collected(self):
+        files_collect = [
+            '/root/curtin-install-cfg.yaml',
+            '/root/curtin-install.log',
+            '/etc/default/grub.d/50-curtin-settings.cfg',
+            '/etc/cloud/cloud.cfg.d/50-curtin-networking.cfg',
+            '/etc/cloud/cloud.cfg.d/curtin-preserve-sources.cfg',
+        ]
+        for file in files_collect:
+            self.assertFileCollected(file)
+
+
+class CurtinScrubbedTest(StageTwoReportTest):
+    """Ensure that curtin conf is picked up and properly scrubbed
+
+    :avocado: tags=stagetwo
+    """
+
+    sos_cmd = '-o curtin'
+    files = [
+        ('curtin-install-cfg.yaml', '/root/curtin-install-cfg.yaml'),
+        ('curtin-install.log', '/root/curtin-install.log'),
+    ]
+
+    keys_obfuscate = [
+        'yaYmBSUuArGt3JYwHU',
+        'WCCtp2JewdWkHEDAW6',
+        'Bm2k3ZdGxFPUK7aZUpTbkRmKQuACKURR',
+    ]
```

Comment:
I know we have similar stance in other tests, but this got me thinking - should we not be checking for the actual scrubbed value instead of (or as well as?) the static strings not being present? I.E. we ensure that `token_key` for example is literally written as `*********`?

Reply:
I've updated the tests, hopefully this would now test for most scenarios, as well as looking at the keys too